dapps/marketplace/src/queries/WechatAuthUrl.js

```diff
@@ -0,0 +1,9 @@
+import gql from 'graphql-tag'
+
+export default gql`
```

Comment:
Should we have a generalized AuthUrl mutation with a param for type?

Reply:
maybe call it attestationAuthUrl?

Reply:
This is file is not in use anymore. I missed to delete it. We have a `getAuthUrl` query with a `provider` param

Reply:
https://github.com/OriginProtocol/origin/blob/135441f649204f176dff32398c8ef8c413552005/dapps/marketplace/src/queries/GetAuthUrl.js#L3-L9